tests/test_box_Box.py

```diff
@@ -193,7 +193,7 @@ def test_vectors(self):
         b_list = [1, 2, 3, 0.1, 0.2, 0.3]
         Lx, Ly, Lz, xy, xz, yz = b_list
         box = freud.box.Box.from_box(b_list)
-        self.assertEqual(
+        npt.assert_allclose(
```

Comment:
Does this actually fail as an equality check from precision issues?

Reply:
@vyasr I changed it so it would match the other tests in the same section, which do have to use `assert_allclose` for precision reasons. We've been getting by on this one, because there isn't any math to throw off the precision of this specific lattice vector, but I think it's better to make them the same.